dqn.cpp

```diff
@@ -22,29 +22,29 @@ void HasBlobSize(caffe::Net<Dtype>& net,
 }
 
 void DQN::LoadTrainedModel(const std::string& model_bin) {
-  net_->CopyTrainedLayersFrom(model_bin);
+  net_actor_->CopyTrainedLayersFrom(model_bin);
```

Comment:
This will need to take two strings -- one for the actor and the other for the critic and load both models.

Either that or we should have a separate method for loading the critic.